frontend/src/metabase/admin/settings/selectors/selectors.js

```diff
@@ -492,6 +493,16 @@ export const ADMIN_SETTINGS_SECTIONS = {
     component: CloudPanel,
     settings: [],
   },
+  whitelabel: {
+    name: t`Appearance`,
+    getHidden: settings => settings["token-features"]?.whitelabel === true,
+    order: 133,
+    component: props => (
+      <UpsellWhitelabel {...props} source="settings-appearance" />
+    ),
+    settings: [],
+    isUpsell: true,
```

Comment:
Note to reviewers:
We will have the "Cloud" sidebar nav marked as the upsell https://linear.app/metabase/issue/CLO-3565/upsells-for-settingscloud

Since we don't have the type/shape strictly defined for a section, I introduced a new optional `isUpsell` key.